Data/option/readme.md

```diff
@@ -0,0 +1,70 @@
+![alt tag](https://raw.githubusercontent.com/QuantConnect/Lean/master/Documentation/logo.white.small.png) 
+## LEAN Data Formats / Options 
+
+QuantConnect hosts options data provided by [AlgoSeek](https://algoseek.com/). The data contains *quotes*, *trades*, and *open interest* data. You can explore options data on our website at https://www.quantconnect.com/data/tree/option/
+
+The data are stored as compressed ZIP files, each containing multiple CSV entries, varying on the option style, e.g. call/put, strike price, and expiration date. 
+
+Options data can be used with the following Resolutions:
+
+* Minute
```

Comment:
We can keep these resolution highlights if you also add it to the equity one. // to make the formats align